### PR TITLE
[Bulk] Validation-05-04-22-metadata-fixes

### PR DESCRIPTION
**Pull request by Roudabeh1348**
This PR fixes 111 issues across multiple categories in azure-docs-sdk-java repo.

### CODE REVIEW

File: docs-ref-services/preview/resourcemanager-databox-readme.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Resource Manager DataBox client library for Java
+description: "Azure Resource Manager DataBox client library for Java - Version 1.0.0-beta.1 contains Microsoft Azure SDK for DataBox Management SDK.  Package tag package-2021-03."
```
Comment (RedZephyr13) [inline docs-ref-services/preview/resourcemanager-databox-readme.md:3]: You have an extra space before the 3rd sentence, which I think can be removed from the description field.  I don' think "Package tag..." helps in any search results.

Comment (Roudabeh1348) [reply docs-ref-services/preview/resourcemanager-databox-readme.md:3]: fixed.

File: docs-ref-services/preview/resourcemanager-databoxedge-readme.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Resource Manager DataBoxEdge client library for Java
+description: "Azure Resource Manager DataBoxEdge client library for Java contains Microsoft Azure SDK for DataBoxEdge Management SDK.  Package tag package-2019-08."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply docs-ref-services/preview/resourcemanager-databoxedge-readme.md:3]: fixed and removed the package tag.

**Comment by Roudabeh1348** [comment]
@sima-zhu @joshfree @scbedd The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. 
Thank you!

**Review by joshfree — CHANGES_REQUESTED** [review]
This PR seems to be updating the same lines as https://github.com/Azure/azure-docs-sdk-java/pull/2324, which I left request for changes on for the same content

**Comment by Roudabeh1348** [comment]
@joshfree I reviewed your comments on PR #2324 and I can go ahead and make those changes as per your instructions. I need some clarifications though, for example, I was not able to find the github/CODEOWNERS file in the azure-sdk-for-java repo,. I would appreciate it, if you could provide me with the URL and I will start make those changes to the author's and mas.author's field, I believe this should eliminate the concern for leaking people's emails on these documents.. I followed the instructions on the spreadsheet and if those instructions need to be changed, we need the URL for that resource so we can update it in the instructions tab of the spreadsheet.  I also wanted to get some clarification on the issue of manager field in the metadata section of the document, if I understood correctly, you wish me to remove that field from the metadata section. I had the same request for another PR from another repo to remove manager tags due to the fact that these articles are old and manager tag is not a reliable source of information.

**Comment by Roudabeh1348** [comment]
repo is autogenerated.

**Comment by joshfree** [comment]
> I was not able to find the github/CODEOWNERS file in the azure-sdk-for-java repo,. I would appreciate it, if you could provide me with the URL 

https://github.com/Azure/azure-sdk-for-java/blob/main/.github/CODEOWNERS

**Pull request closed without merging**